sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/Program.cs

```diff
@@ -80,25 +82,18 @@ public static async Task Main(string[] args)
         }
 
         /// <summary>
-        /// Illustrates how to construct a <see cref="DigitalTwinsClient"/>, using the <see cref="ClientSecretCredential"/>
+        /// Illustrates how to construct a <see cref="DigitalTwinsClient"/>, using the <see cref="DefaultAzureCredential"/>
         /// implementation of <see cref="Azure.Core.TokenCredential"/>.
-        /// </summary>
-        /// <param name="tenantId">The Id of the tenant of the application Id.</param>
-        /// <param name="clientId">The application Id.</param>
-        /// <param name="clientSecret">A client secret for the application Id.</param>
+        /// </summary>        
         /// <param name="adtEndpoint">The endpoint of the digital twins instance.</param>
-        private static DigitalTwinsClient GetDigitalTwinsClient(string tenantId, string clientId, string clientSecret, string adtEndpoint)
-        {
+        private static DigitalTwinsClient GetDigitalTwinsClient(string adtEndpoint)
+        {   
             #region Snippet:DigitalTwinsSampleCreateServiceClientWithClientSecret
 
-            // By using the ClientSecretCredential, a specified application Id can login using a
+            // By using the DefaultAzureCredential, a specified application Id can login using a
```

Comment:
this comment still not exactly accurate any more. Yes, an appId can login with a client secret, but there are other ways of logging in that is possible.

Perhaps we should change our parameters such that all the auth values are optional. If they are present, we override environment variables.

Then we probably could go to just one way of instantiating the client, because DefaultAzureCredential works as client secret credential and interactive browser credential.

Reply:
I thought we did not want to put so much details here and tried to keep it less detailed. I can add all the details if you think that would be good to add. Also I've not been able to get this working for interactive login yet. Will be easier to make that change when I get that working. I'll probably just make a PR once I get that to work as well.